tiledb/common/common.h

```diff
@@ -74,4 +74,9 @@ constexpr bool is_experimental_build = true;
 constexpr bool is_experimental_build = false;
 #endif  // TILEDB_EXPERIMENTAL_FEATURES
 
+/*
+ * Platform/machine config
+ */
+#include "platform.h"
```

Comment:
These should not go `common.h`. We want to isolate platform-dependent code, not commingle it with everything. It should *not* be the ordinary case the platform-specific code is sprinkled throughout everything.

Reply:
fixed